content-repo/gendocs.py

```diff
@@ -335,7 +335,7 @@ def process_readme_doc(target_dir: str, content_dir: str, prefix: str,
             f.write(content)
         return doc_info
     except Exception as ex:
-        print(f'fail: {readme_file}. Exception: {traceback.format_exc()}')
+        print(f'fail: {os.path.normpath(readme_file)}. Exception: {traceback.format_exc()}')
```

Comment:
@MosheEichler although it fixes the issue. i think the best solution here is to normalize the path when it's being received from the argparser. from there is will propagate correctly to all the places it's being used. 

I think this is the correct path to do it once, for both the dir and target dir
https://github.com/demisto/content-docs/pull/1342/files#diff-386670aa2981f93d7f2638b911246e4097516bbdea71a8dfcd1acc89374de54bL897